### DISCUSSION TITLE
ImportError: cannot import name 'get_default_executor'

### DISCUSSION THREAD

**Discussion by picousse** [apache/airflow#13771] [Q&A] [answered]
Hi,
I just went to the hassle of upgrading airflow to 2.0.

I was using a celery executor.

When running the scheduler I now get:

```
Jan 19 19:33:58 airflow bash[1145]:   File "/usr/local/bin/airflow", line 26, in <module>
Jan 19 19:33:58 airflow bash[1145]:     from airflow.bin.cli import CLIFactory
Jan 19 19:33:58 airflow bash[1145]:   File "/home/airflow/.local/lib/python3.6/site-packages/airflow/bin/cli.py", line 70, in <module>
Jan 19 19:33:58 airflow bash[1145]:     from airflow.executors import get_default_executor
Jan 19 19:33:58 airflow bash[1145]: ImportError: cannot import name 'get_default_executor'
```

I installed using pip3 install apache-airflow[celery].

What am I doing wrong?

**Comment by potiuk** [comment, marked as answer]
I think you installed airflow 1.10 differently before. In Airflow 1.10 we had indeed "airfow" script but we are using console script now, so there should bot be /usr/local/bin/airflow script - console script works differently `https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html`. This script should be uninstalled if you install airfllow 1.10 (which happens automatically when you install 2.0) - apparently you had locally modified installation or airflow installed to /usr/local/bin and it remained there. 

Possibly removal of the script from /usr/local/bin will work, but you might have much bigger problems if your environment is manually modified. I'd recommend reinstalling it from scratch or some heavy cleanup (but you need to know what you are doing)

**Comment by picousse** [comment]
Okay, so I 'd better restart install everything :-s.

However it is still a bit unclear how I should actually do this?
On https://airflow.apache.org/docs/apache-airflow/stable/installation.html, the examples cover 1.10.xx versions, not 2.0.0?

Should use that example and change the version? or should I install it using pip?

Thanks for the help!

**Comment by potiuk** [comment]
Yes. And Yes.

Here you can see updated version of that page (with 2.0 in) which will be released as part of 2.0.1
https://github.com/apache/airflow/blob/master/docs/apache-airflow/installation.rst

**Comment by picousse** [comment]
Hi,
Super. Thanks for the super fast and helpful replies!

Pieter
